tests/ui/stub-attribute/attribute.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+//! Checks that the `kani::stub` attribute is accepted
+
+fn foo() {}
+
+fn bar() {}
+
```

Comment:
Since you don't do any checking here, you don't actually need these functions to be defined, right?